### ISSUE TITLE
Execute Callbback Action is not working

### ISSUE THREAD

**Issue by ImadBouirmane** [FlutterFlow/flutterflow-issues#2684]
### Has your issue been reported?

- [X] I have searched the existing issues and confirm it has not been reported.
- [X] I give permission for members of the FlutterFlow team to access and test my project for the sole purpose of investigating this issue.

### Current Behavior

The action of deletion for firestore backend call is not working based on **Excute Callback action in a component**

### Expected Behavior

Delete the specified document from the list.

### Steps to Reproduce

1- Create a component and pass an action
2- Set this parameter as event for a widget like button 
3- Pass the parameter in a page use this component for something trigerred 

### Reproducible from Blank

- [X] The steps to reproduce above start from a blank project.

### Bug Report Code (Required)

ITESj/KAvJdgoupK17rAdMJQhQAkH1x/bZJEl9caZxA0B4DwPLAAYur+VFpVUsuKYGNmKGeZnTsH+PDrvvGeU/Y5ISebQoNo+rtxWwn/RkehaJC1Daq7P0B9EeRhf36506uO2gpvBNJZWVVnw3z/cq3qNleeY8aSfxBlZ7vfcPo=

### Context

The action of deletion for firestore backend call is not working based on **Excute Callback action in a component** 
I need to let user if he want to delete an item from list will by confirmation bottom sheet.

### Visual documentation

![image](https://github.com/FlutterFlow/flutterflow-issues/assets/8468212/69f2fd96-81c4-4dd7-983e-499872649fe0)
![image](https://github.com/FlutterFlow/flutterflow-issues/assets/8468212/dd9e1e6f-39e5-45ed-a873-ad628794cbc6)
![image](https://github.com/FlutterFlow/flutterflow-issues/assets/8468212/b0d8030c-251a-4372-a61a-e94483724e96)

### Additional Info

![image](https://github.com/FlutterFlow/flutterflow-issues/assets/8468212/b0cb608f-1abb-423e-a4cc-9c917ca26b90)
![image](https://github.com/FlutterFlow/flutterflow-issues/assets/8468212/3cf400e5-f3c6-4b81-b381-d9473d8281b0)

### Environment

```markdown
- FlutterFlow version: v4.1.38+
- Platform: Windows
- Browser name and version:
- Operating system and version affected:  Win 11 NT
```

**Comment by ignalauret** [comment]
Hey @ImadBouirmane thanks for your report. I just replicated your steps on a blank project, and I was able to correctly execute the action passed to a Component and used on a button inside that component with "Execute Callback". 
So I went into your project to see if you have some misconfiguration, but it's really big and I couldn't find what you are showing there. 
I encourage you to contact our customer support, they will help you implement what you are looking for. I'm closing this issue because I don't see a bug here. 
Have a great day.
